ignite/cmd/account_import.go

```diff
@@ -34,6 +34,9 @@ func accountImportHandler(cmd *cobra.Command, args []string) error {
 		secret, _ = cmd.Flags().GetString(flagSecret)
 	)
 
+	session := cliui.New(cliui.StartSpinner())
```

Comment:
The spinner should not start when using `cliquiz` directly:

```suggestion
	session := cliui.New()
```

An alternative would be to use `session.Ask()` which should be the way to go within a session context but the `getPassphrase()` would require a session argument in that case because internally it is also using `cliquiz`.

Reply:
Ah, i see. Let's just revert